### ISSUE TITLE
how to finetune the pretrained model in pytorch?

### ISSUE THREAD

**Issue by visonpon** [junyanz/pytorch-CycleGAN-and-pix2pix#85]
@phillipi @alanyee  i want to use my own datasets to finetune the trained model, can you give me some advice ? thanks~

**Comment by junyanz** [comment]
To fine-tune a pre-trained model, or resume the previous training, use the `--continue_train` flag. The program will then load the model based on `which_epoch`. By default, the program will initialize the epoch count as 1. Set `--epoch_count <int>` to specify a different starting epoch count.

**Comment by visonpon** [comment]
@junyanz when use the trained model like this:
`python train.py --dataroot */data/ --save_latest_freq **10000[my own dataset size]** --niter 10 --niter_decay 10 --dataset_model unaligned --continue_train  --epoch_count 10 --which_epoch */experimnet_name/10`

the trainning log file show that:
epoch 10 itert:-100000[seemsl like 10000*10???]  loss: 

is this normal?thanks~

**Comment by visonpon** [comment]
@junyanz but i dont see "--which_epoch" in the train.py

**Comment by junyanz** [comment]
See [here](https://github.com/junyanz/pytorch-CycleGAN-and-pix2pix/blob/master/options/train_options.py#L16).
